pkgs/top-level/all-packages.nix

```diff
@@ -10359,6 +10359,8 @@ with pkgs;
     };
 
   ndi = callPackage ../development/libraries/ndi { };
+  
+  nemodex-usb-flasher = callPackage ../pkgs/nemodex-usb-flasher {};
```

Comment:
Do not modify this file. Putting your definition in by-name handles it automatically.